### PR TITLE
Add a GPU longrun with 1M microphysics (aquaplanet grid mean)

### PR DESCRIPTION
**Pull request by trontrytel**
@szy21 could you double check I didn't mess up in the pipeline setup?

Also, what are the next configurations I should add 1M to?

### CODE REVIEW

File: src/parameterized_tendencies/microphysics/precipitation.jl [hunk 1]
```diff
@@ -190,7 +190,7 @@ function qₚ(ρqₚ::FT, ρ::FT) where {FT}
 end
 
 function limit(q::FT, dt::FT) where {FT}
-    return q / dt
+    return q / dt / FT(5)
```
Comment (szy21) [inline src/parameterized_tendencies/microphysics/precipitation.jl:193]:
```suggestion
    return q / dt / 5
```

File: src/prognostic_equations/hyperdiffusion.jl [hunk 2]
```diff
@@ -246,7 +246,7 @@ NVTX.@annotate function apply_tracer_hyperdiffusion_tendency!(Yₜ, Y, p, t)
     # triggers allocations.
     for (ᶜρχₜ, ᶜ∇²χ, χ_name) in matching_subfields(Yₜ.c, ᶜ∇²specific_tracers)
         ν₄_scalar =
-            ifelse(χ_name in (:q_rai, :q_sno), 0.1 * ν₄_scalar, ν₄_scalar)
+            ifelse(χ_name in (:q_rai, :q_sno), FT(0) * ν₄_scalar, ν₄_scalar)
```
Comment (szy21) [inline src/prognostic_equations/hyperdiffusion.jl:249]:
```suggestion
            ifelse(χ_name in (:q_rai, :q_sno), 0 * ν₄_scalar, ν₄_scalar)
```

File: config/longrun_configs/longrun_aquaplanet_clearsky_1M.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+rayleigh_sponge: true
+moist: "equil"
+precip_model: "1M"
+surface_setup: "DefaultMoninObukhov"
+vert_diff: "FriersonDiffusion"
+implicit_diffusion: true
+approximate_linear_solve_iters: 2
+rad: "clearsky"
+dt_rad: "6hours"
+job_id: "gpu_aquaplanet_clearsky_1M"
```
Comment (szy21) [inline config/longrun_configs/longrun_aquaplanet_clearsky_1M.yml:17]:
```suggestion
job_id: "longrun_aquaplanet_clearsky_1M"
```
(This needs to be the same as JOB_NAME in the pipeline for the artifacts to be uploaded. I usually make it the same as the config file name. Also remember to add it to ci_plots and longrun docs.)

**Review by szy21 — APPROVED** [review]

**Comment by szy21** [comment]
The next would be all the `gpu_aquaplanet_dyamond*.yml` in `config/gpu_pipeline`, after https://github.com/CliMA/ClimaAtmos.jl/pull/2792 is merged. I can point you to the amip setup in coupler next week.

File: config/longrun_configs/longrun_aquaplanet_clearsky_1M.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+dt: "120secs"
+t_end: "100days"
+h_elem: 16
+z_elem: 63
+dz_bottom: 30.0
+dz_top: 3000.0
+z_max: 55000.0
+rayleigh_sponge: true
+moist: "equil"
```
Comment (akshaysridhar) [inline config/longrun_configs/longrun_aquaplanet_clearsky_1M.yml:9]: @trontrytel Could you please also update the `default_config.yml` helper for the `moist` flag - it currently says `non_equil` is a valid option, but the correct string option should be `nonequil` based on the `model_getters.jl` file.

**Pull request merged**